MAUI/ImageEditor/zooming.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: Zooming in .NET MAUI ImageEditor Control | Syncfusion
```

Comment:
title: Zoom image using .NET MAUI Image Editor Control | Syncfusion
description: Learn here all about how to zoom image in Syncfusion .NET MAUI Image Editor(SfImageEditor) control and more.

Reply:
Updated a title and description